controls/SV-204438.rb

```diff
@@ -19,41 +18,39 @@
     Enter password:
     Confirm password:'
   impact 0.7
-  tag 'legacy': ['SV-95717', 'V-81005']
-  tag 'severity': 'high'
-  tag 'gtitle': 'SRG-OS-000080-GPOS-00048'
-  tag 'gid': 'V-204438'
-  tag 'rid': 'SV-204438r744095_rule'
-  tag 'stig_id': 'RHEL-07-010482'
-  tag 'fix_id': 'F-4562r744094_fix'
-  tag 'cci': ['CCI-000213']
+  tag legacy: ['SV-95717', 'V-81005']
+  tag severity: 'high'
+  tag gtitle: 'SRG-OS-000080-GPOS-00048'
+  tag gid: 'V-204438'
+  tag rid: 'SV-204438r744095_rule'
+  tag stig_id: 'RHEL-07-010482'
+  tag fix_id: 'F-4562r744094_fix'
+  tag cci: ['CCI-000213']
   tag nist: ['AC-3']
-  tag subsystems: ["boot","bios"]
+  tag subsystems: ['boot', 'bios']
   tag 'host'
 
   if virtualization.system.eql?('docker')
```

Comment:
<img width="795" alt="image" src="https://github.com/mitre/redhat-enterprise-linux-7-stig-baseline/assets/2015821/654c7f6d-6812-4a0f-aca9-2297c71d1218">

idk if there is a 'virtualization.system' for determining if it's an ec2 box.  fairly sure the box i spun up is more or less a vm as opposed to anything raw metal.

regardless, we should probably have an explicit check if the file is there and make a fail/skip decision accordingly as opposed to letting it skip out like this unintentionally.

for your debugging purposes, this happened on both of the following AMIs running on t2.micros using the default ec2.inputs.yml that's bundled in the repo.

https://aws.amazon.com/marketplace/pp/prodview-4rvgwodhco57y?sr=0-1&ref_=beagle&applicationId=AWSMPContessa

https://aws.amazon.com/marketplace/pp/prodview-5ydzzv4pbudbm?sr=0-7&ref_=beagle&applicationId=AWSMPContessa

Reply:
I think this is an appropriate response because the file was not found and therefore the control is not going to give accurate results unless something is resolved to find the file, for example:
- setting the input('grub_user_boot_files') to the correct value if it is not the default for the system
- running the profile as root (--sudo) so the files are accessible